### DISCUSSION TITLE
autocomplete-python traceback output

### DISCUSSION THREAD

**Discussion by ghost** [atom/atom#22269] [General]
Getting this thing when I type a letter or a variable 
**_Traceback (most recent call last):
  File "C:\Users\harim\.atom\packages\autocomplete-python\lib\completion.py", line 8, in <module>_**

![Screenshot (2)](https://user-images.githubusercontent.com/68155163/115910035-f9999980-a489-11eb-9197-d5067052fe79.png)

**Comment by ThatXliner** [comment]
You should put this in an issue

**Reply by ghost** [reply]
okay thank you
